.github/workflows/deploy.yml

```
@@ -0,0 +1,26 @@
+name: Deploy Production
+
+on:
+  push:
+    branches:
+      - master
```

This should be main, correct?

```suggestion
      - main
```

---

Oh, whoops! Fixed that. I copied the deploy script from constants and forgot to change that.